package.json

```diff
@@ -83,7 +83,8 @@
     "**/jsprim/json-schema": "^0.4.0",
     "**/@storybook/**/yuidocjs/markdown-it": "^12.3.2",
     "**/ember-cli/markdown-it-terminal/markdown-it": "^12.3.2",
-    "**/ember-cli-mirage/@embroider/macros": "^1.0.0"
+    "**/ember-cli-mirage/@embroider/macros": "^1.0.0",
+    "**/ember-arg-types/prop-types": "^15.8.1"
```

Comment:
@cameronperera 
None blocking comment: if `ember-arg-types` is used only within `addons/rose` (needs to be check), I would suggest to add the `rose` within the resolution, just to narrow it down there and avoid possible issues. I.E: `**/rose/**/ember-arg-types/prop-types`.